server/base/src/main/java/org/apache/accumulo/server/util/serviceStatus/ServiceStatusReport.java

```diff
@@ -38,18 +39,21 @@ public class ServiceStatusReport {
 
   private static final Logger LOG = LoggerFactory.getLogger(ServiceStatusReport.class);
 
-  private static final Gson gson = new Gson();
```

Comment:
Did you search the code for uses of gson that did not use the singleton?  Or did you just happen to find this?

Reply:
I searched for all uses of gson to update and I noticed this so I switched to the singleton